src/components/Fetch_API_Data.js

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+
+const FetchAPIData = () => {
+  const [data, setData] = useState([]);
```

Comment:
- [ ] This is the project requirements 👇🏻 Kindly make sure to create A `<Quotes>` component in which you will implement the fetchData  to the API with useEffect hook.

![image](https://user-images.githubusercontent.com/79058364/227741497-f9e43c89-c0f4-4f7a-b397-4f1f05051f5f.png)

- 🤔 I think you only need to rename your fetch_API_Date.js file. 


```jsx
function Quotes() {
  const [quotes, setQuotes] = useState([])
  .....

  useEffect(() => {
    const fetchQuotes = async () => {
      setIsLoading(true)
      try {
        const res = await fetch(
          "https://api.api-ninjas.com/v1/quotes?category=happiness",
        .......
  
    fetchQuotes()
  }, [setQuotes])

  return (
    ....
  )
}

export default Quotes
```